# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: tools/lint/eslint/eslint-plugin-mozilla/lib/environments/privileged.js
# Commit: ca737b661c6e
# Full Hash: ca737b661c6eeaecfebf6f2bd27aca46ef656ae4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-08-23 04:21:33
# Description:
#   Bug 1849271 - Flip test when checking for the presence of optional colorspace members on a VideoDecoderConfig. r=media-playback-reviewers,alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D186530
# ==============================================================================

diff -r bc91ede31572 -r ca737b661c6e tools/lint/eslint/eslint-plugin-mozilla/lib/environments/privileged.js
--- a/tools/lint/eslint/eslint-plugin-mozilla/lib/environments/privileged.js	Tue Aug 22 14:03:58 2023 +0000
+++ b/tools/lint/eslint/eslint-plugin-mozilla/lib/environments/privileged.js	Tue Aug 22 14:33:21 2023 +0000
@@ -36,7 +36,10 @@
     AudioBuffer: false,
     AudioBufferSourceNode: false,
     AudioContext: false,
+    AudioDecoder: false,
     AudioDestinationNode: false,
+    AudioData: false,
+    AudioEncoder: false,
     AudioListener: false,
     AudioNode: false,
     AudioParam: false,
@@ -162,6 +165,8 @@
     DragEvent: false,
     DynamicsCompressorNode: false,
     Element: false,
+    EncodedAudioChunk: false,
+    EncodedVideoChunk: false,
     ErrorEvent: false,
     Event: false,
     EventSource: false,
@@ -704,6 +709,10 @@
     VTTCue: false,
     VTTRegion: false,
     ValidityState: false,
+    VideoColorSpace: false,
+    VideoDecoder: false,
+    VideoEncoder: false,
+    VideoFrame: false,
     VideoPlaybackQuality: false,
     VideoTrack: false,
     VideoTrackList: false,
